app/controllers/idv/enter_password_controller.rb

```diff
@@ -123,8 +123,15 @@ def gpo_mail_service
       @gpo_mail_service ||= Idv::GpoMail.new(current_user)
     end
 
+    def is_enhanced_ipp?
+      return false if !defined?(resolved_authn_context_result)
+      return resolved_authn_context_result.enhanced_ipp?
+    end
+
     def init_profile
-      idv_session.create_profile_from_applicant_with_password(password)
+      is_enhanced_ipp = is_enhanced_ipp?
+
+      idv_session.create_profile_from_applicant_with_password(password, is_enhanced_ipp)
```

Comment:
seems easy to inline this method call
```suggestion
      idv_session.create_profile_from_applicant_with_password(password, is_enhanced_ipp?)
```

Reply:
Done. I passed in `resolved_authn_context_result.enhanced_ipp?` -- same pattern as Eileen in [PR# 10820](https://github.com/18F/identity-idp/pull/10820/files#diff-56600fea988ec9b77006528a7fd8d1b8c75f80a3689d0077cfb04d20bf903767)